tests/integration/test_tickets.py

```diff
@@ -55,7 +56,7 @@ def test_tickets_POST(client, auth_headers):
     newTicket = {
         'sender': 1,
         'tenant': 1,
-        'status': 'new',
+        'status': TicketStatus.New,
```

Comment:
for the payload on the test we should use a hard-coded string here, because that is what we would get from a request.
```suggestion
        'status': 'New',
```